src/stripe.ios.ts

```diff
@@ -1,4 +1,6 @@
 import { View } from 'tns-core-modules/ui/core/view';
+import { topmost } from "tns-core-modules/ui/frame";
```

Comment:
I believe this is now in master. You may need to rebase.

Reply:
Just check and it's not actually